ext/opcache/Optimizer/sccp.c

```diff
@@ -844,13 +844,20 @@ static inline int ct_eval_func_call(
 				|| zend_string_equals_literal(name, "urldecode")
 				|| zend_string_equals_literal(name, "rawurlencode")
 				|| zend_string_equals_literal(name, "rawurldecode")
-				|| zend_string_equals_literal(name, "php_uname")) {
+				|| zend_string_equals_literal(name, "php_uname")
+				|| zend_string_equals_literal(name, "dirname")
+				|| zend_string_equals_literal(name, "basename")
```

Comment:
Unfortunately, basename is locale-sensitive, so it cannot be evaluated here.

Reply:
true, I mussed the mb ones, removed this ones :+1: